docs/governance/canonical-reconciliation-register.md

```
@@ -209,6 +214,8 @@ ADR-0020, ADR-0021, ADR-0022, ADR-0023, ADR-0024, ADR-0025, and ADR-0026
 respectively.
 #14 is separately resolved by ADR-0027 without promoting concrete external-format
 adapters into the current milestone.
+#17 is separately resolved by ADR-0028 without stabilizing a general plugin
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)